intelmq/bots/outputs/esm/output_ip.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+"""
+
+ESMOutputBot connects to McAfee Enterprise Security Manager, and updates IP based watchlists
+
+Parameters:
+esm_ip: IP Address of ESM
+esm_user: username to connect to ESM
+esm_pw: Password of esm_user
+esm_watchlist: Destination watchlist to update
+field: field from intelMQ message to extract (e.g. destination.ip)
+
+"""
+
+from intelmq.lib.bot import Bot
+
+import json
+
+try:
+    from mfe_saw.esm import ESM
+except ImportError:
+    ESM = None
+
+
+class ESMIPOutputBot(Bot):
+
+    def init(self):
+        if ESM is None:
+            self.logger.error('Could not import mfe_saw. Please install it.')
+            self.stop()
```

Comment:
please use a `raise ValueError('Could not...')` for these two lines instead (I just fixed this in other places too)